### PR TITLE
Add L2 Testnet Deployments

### PR DESCRIPTION
**Pull request by arjun-io**
[File Filter](https://github.com/equilibria-xyz/perennial-mono/pull/114/files?file-filters%5B%5D=.sh&file-filters%5B%5D=.ts&show-viewed-files=true) to exclude JSON files for easier review

### CODE REVIEW

File: packages/perennial-examples/package.json [hunk 1]
```diff
@@ -35,7 +37,7 @@
   "license": "APACHE-2.0",
   "dependencies": {
     "@equilibria/perennial": "0.2.1",
-    "@equilibria/root": "0.1.6",
+    "@equilibria/root": "0.2.0",
```
Comment (kbrizzle) [inline packages/perennial-examples/package.json:40]: nit: do we need this for deployments specifically? otherwise might be easier to keep track of in another PR

**Review by kbrizzle — APPROVED** [review]

File: packages/perennial-examples/package.json (hunk 1 above)
Comment (arjun-io) [reply packages/perennial-examples/package.json:40]: We need the new version for Perennial Core (crosschain owner) so I just upgraded all the packages to keep them consistent
